dockerfiles/run/guice/cassandra-ldap/destination/conf/blob.properties

```diff
@@ -1,10 +1,9 @@
-# ============================================ Blobs Exporting ==============================================
-# Read https://james.apache.org/server/config-blob-export.html for further details
+# ============================================= BlobStore Implementation ==================================
+# Read https://james.apache.org/server/config-blobstore.html for further details
 
-# Choosing blob exporting mechanism, allowed mechanism are: localFile, linshare
-# LinShare is a file sharing service, will be explained in the below section
-# Optional, default is localFile
-blob.export.implementation=localFile
+# Choose your BlobStore implementation
+# Mandatory, allowed values are: cassandra, s3
+implementation=cassandra
```

Comment:
Isn't the implementation always Cassandra in cassandra-guice product ?

Reply:
I can only speak for the openpaas image. But if no blob.properties is provided james start successfully, if a blob.properties is provided without 'implementation' it fails with a message pointing out this problem.